### DISCUSSION TITLE
Suggestion: convert the usage of tkinter constants to strings

### DISCUSSION THREAD

**Discussion by Moosems** [insolor/tkinter-layout-helpers#13] [General]
I would suggest changing the tkinter constants to strings. Example: `tk.SW` -> "sw". It makes the code look a little cleaner and there's no harm to changing them. If you prefer using the constants, go ahead, it works either way :D.

**Comment by insolor** [comment]
@Moosems, in general, I prefer to use constants (even for flake8-tkinter my original idea was to enforce using constants: https://github.com/rdbende/flake8-tkinter/blob/798501c613c09776bf27950158a4b69d5e90229c/README.md). Even more, I'd prefer if there were enums instead of constants, and type annotations for parameters in the tkinter code, so I would know which values are possible without the need to search for documentation or examples of code.

So, no, I'm not changing constants to string literals.

**Reply by Moosems** [reply]
Makes sense. I really like the project by the way, it looks pretty promising :).

**Reply by insolor** [reply]
Thank you, I appreciate it :)

**Reply by Moosems** [reply]
Is there any way I could help out with it? I could try writing some basic docs but I might not know it well enough to do anything in depth 🙁.

**Reply by insolor** [reply]
I'm not sure about “real” documentation, although at least docstrings would be helpful. I'd suggest you to experiment with the package and then maybe you'll have some ideas how to improve it.
